### DISCUSSION TITLE
Blog post date-only format

### DISCUSSION THREAD

**Discussion by brabster** [squidfunk/mkdocs-material#6998] [Ideas]
I notice that blog post dates have a time/timezone element, but it always shows midnight UTC for my content (and the mkdocs-material blog)

![image](https://github.com/squidfunk/mkdocs-material/assets/38702/d780431d-b58e-4856-b2d9-de546052e428)

![image](https://github.com/squidfunk/mkdocs-material/assets/38702/2859c941-6a85-4488-8f2e-c2460ebd8da3)

Seems a bit unnecessarily cluttered - I can't see a way to tell mkdocs-material to just format as a date. Maybe I missed an option?

**Comment by alexvoss** [comment]
A very similar question was asked just yesterday: https://github.com/squidfunk/mkdocs-material/discussions/6996. The [documentation](https://squidfunk.github.io/mkdocs-material/plugins/blog#config.post_date_format) suggests that you can create your own format using the [Babel pattern ](https://babel.pocoo.org/en/latest/dates.html#pattern-syntax).

Generally, it is often a good idea to search the forum as it does contain a lot of answers already.

**Reply by squidfunk** [reply]
Additionally, this is not the default for [`post_date_format`](https://squidfunk.github.io/mkdocs-material/plugins/blog/#config.post_date_format), so it was already changed to another value. The default is `long`, resulting in locale-aware formatted dates like "January 31, 2024", so it sounds like the documentation was already consulted.

**Reply by alexvoss** [reply]
@squidfunk, I also get a date and time for posts:

![image](https://github.com/squidfunk/mkdocs-material/assets/4134224/8f4afa35-5773-4617-82ff-548b93044c4e)

Reproduction:

[9.5.16+insiders.4.53.4-datetime-blog.zip](https://github.com/squidfunk/mkdocs-material/files/14823534/9.5.16%2Binsiders.4.53.4-datetime-blog.zip)

**Reply by squidfunk** [reply]
Please create a new issue then.

**Reply by alexvoss** [reply]
Posted: https://github.com/squidfunk/mkdocs-material/issues/7001 Seems to be a recent regression.

**Comment by brabster** [comment]
I hadn't set anything explicitly for date formatting - as far as I can tell the full date and and time as in my screenshots is what you get by default - mkdocs section here https://github.com/brabster/tw-site-mkdocs/blob/a85f9d83084196f33e4098016357081fb8a78a2b/mkdocs.yml#L31

I got confused by  https://squidfunk.github.io/mkdocs-material/plugins/blog/#config.post_date_format - the examples for the shortcodes suggested that I'd get a date, but I was seeing a datetime - which is why I asked the question.

If I do override the format then I can make it just be the date https://github.com/brabster/tw-site-mkdocs/blob/5df674e90a11e964114d2082d71e2d49b05a95ff/mkdocs.yml#L34 - all working for me now
